lib/experimental/Layouts/Utils/DetailsItemsList/index.stories.tsx

```diff
@@ -17,8 +17,8 @@ const weekdays = (
   />
 )
 const teams = (
-  <BadgesList
-    badges={[
+  <TagsList
+    tags={[
       { text: "Management", avatar: { alt: "M" } },
       { text: "Engineering", avatar: { alt: "E" } },
       { text: "Managers", avatar: { alt: "M" } },
```

Comment:
Why not just taking the first letter of the `text` prop? I see `avatar->alt` kind of redundant

Reply:
I think I remember cases where they took the first two letters? If that doesn't happen they yeah, I agree

Reply:
maybe take the first one if `alt` is not there to give a meaningful default